tests/components/test.pay-method-choice.jsx

```diff
@@ -123,29 +126,73 @@ describe('Pay Method Choice', function() {
 
 });
 
-describe('Single Card Choice', function() {
-
-    var payMethodData = [{
-        'id': 1,
-        'resource_uri': '/braintree/mozilla/paymethod/1/',
-        'truncated_id': '4444',
-        'type_name': 'MasterCard',
-       },
-     ];
-
-    beforeEach(function() {
-      this.PayMethodChoice = TestUtils.renderIntoDocument(
-        <PayMethodChoice payMethods={payMethodData} />
-      );
-    });
-
-    it('Has the card selected when just one card', function() {
-        var input = helpers.findByTag(this.PayMethodChoice, 'input');
-        assert.equal(input.getDOMNode().getAttribute('checked'), '');
-    });
-
-    it('Has the submit button enabled when just one card', function() {
-      var submitButton = helpers.findByTag(this.PayMethodChoice, 'button');
-      assert.equal(submitButton.getDOMNode().getAttribute('disabled'), null);
-    });
+
+describe('Single Pay Method (List)', function() {
+
+  beforeEach(function() {
+    this.PayMethodChoice = TestUtils.renderIntoDocument(
+      <PayMethodChoice payMethods={singlePayMethodData} />
+    );
+  });
+
+  it('has the card selected with a single pay method', function() {
+      var input = helpers.findByTag(this.PayMethodChoice, 'input');
+      assert.equal(input.getDOMNode().getAttribute('checked'), '');
+  });
+
+  it('has the submit button enabled with a single pay method', function() {
+    var submitButton = helpers.findByTag(this.PayMethodChoice, 'button');
+    assert.equal(submitButton.getDOMNode().getAttribute('disabled'), null);
+  });
+
+});
+
+
+describe('Pay Method Choice (DropDown)', function() {
+
+  beforeEach(function() {
+    this.PayMethodChoice = TestUtils.renderIntoDocument(
+      <PayMethodChoice
+        payMethods={payMethodData}
+        useDropDown={true}
+      />
+    );
+  });
+
+  it('displays a drop-down with 6 options', function() {
+    assert.equal(
+      helpers.findAllByTag(this.PayMethodChoice, 'select').length, 1);
+    assert.equal(
+      helpers.findAllByTag(this.PayMethodChoice, 'option').length, 6);
```

Comment:
this would be more maintainable if it were `payMethodData.length` instead of `6`